recipes/warpx/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "warpx" %}
+{% set version = "20.11" %}
+{% set build = 0 %}
+{% set sha256 = "e0e8d9ef9ecb53d2d0f5091939088a538863bc3779ddfeab159b5a51279f399c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ECP-WarpX/WarpX/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  run_exports:
+    # patch-releases are not ABI compatible
+    - {{ pin_subpackage('warpx', min_pin='x.x.x', max_pin='x.x.x') }}
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make          # [unix]
+    - cmake >=3.14.0
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx]
+  host:
+    - boost-cpp >=1.66.0
```

To use global conda-forge pinning
```suggestion
    - boost-cpp
```